css/_sidenavs.scss

```diff
@@ -110,3 +110,14 @@
     margin-right: 10px!important;
     margin-left: 5px!important;
 }
+
+md-sidenav {
```

Comment:
In addition, I found the background color will be transparent, if I remove all Angular Material CSS rules from the page. It seems we should define a default background color for some elements.